library/tiqr/Tiqr/Message/GCM.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * This file is part of the tiqr project.
+ * 
+ * The tiqr project aims to provide an open implementation for 
+ * authentication using mobile devices. It was initiated by 
+ * SURFnet and developed by Egeniq.
+ *
+ * More information: http://www.tiqr.org
+ *
+ * @author Peter Verhage <peter@egeniq.com>
+ * 
+ * @package tiqr
+ *
+ * @license New BSD License - See LICENSE file for details.
+ *
+ * @copyright (C) 2010-2011 SURFnet BV
+ */
+
+
+/** @internal base includes */
+require_once('Tiqr/Message/Abstract.php');
+
+//require_once('Zend/Gdata/ClientLogin.php');
```

Comment:
Should be either removed (if it's no longer needed) or commented (if there is a reason it's disabled). Commented code without comments (cocwoc) is a source of technical debt.